src/components/Books.js

```diff
@@ -0,0 +1,91 @@
+import { PropTypes } from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { Progress } from 'antd';
+import { removeBook } from '../redux/books/books';
+
+const Books = ({
+  id, title, author, category,
+}) => {
+  const percent = Math.floor((Math.random() * 99) + 1);
+  const chapter = Math.floor((Math.random() * 25) + 1);
+  const dispatch = useDispatch();
+  const removeHandler = () => {
+    dispatch(removeBook(id));
+  };
+  return (
+
+    <div className="book">
+      <div className="content">
+        <div className="info">
+          <h4 className="category">{category}</h4>
+          <h2 className="title">{title}</h2>
+          <h6 className="author">{author}</h6>
+          <div className="action-buttons">
+            <button className="button-outline" type="button">
+              Comments
+            </button>
+            <div className="vertical-divider" />
+            <button
+              className="button-outline"
+              type="button"
+              onClick={removeHandler}
+            >
+              Remove
+            </button>
+            <div className="vertical-divider" />
+            <button className="button-outline" type="button">
+              Edit
+            </button>
```

Comment:
- [ ] As suggested by the previous reviewer, Kindly makes sure you stick to the original design as much as possible. The spacing between your element(`margin, padding`) should be similar to the original template.

Your design :point_down: 
![Screenshot from 2023-02-11 20-52-56](https://user-images.githubusercontent.com/97100091/218276336-912cc044-3b14-41bf-ae0a-07c61d562532.png)